extensions/arc/src/ui/dashboards/miaa/miaaDashboardOverviewPage.ts

```diff
@@ -151,4 +196,80 @@ export class MiaaDashboardOverviewPage extends DashboardPage {
 		).component();
 	}
 
+	private async handleRegistrationsUpdated(): Promise<void> {
+		const reg = this._controllerModel.getRegistration(ResourceType.sqlManagedInstances, this._miaaModel.namespace, this._miaaModel.name);
+		if (reg) {
+			this._instanceProperties.resourceGroup = reg.resourceGroupName || '-';
+			this._instanceProperties.dataController = this._controllerModel.controllerRegistration?.instanceName || '-';
+			this._instanceProperties.region = (await getAzurecoreApi()).getRegionDisplayName(reg.location);
+			this._instanceProperties.subscriptionId = reg.subscriptionId || '-';
+			this._instanceProperties.computeAndStorage = reg.vCores || '-';
+			this._instanceProperties.host = reg.externalEndpoint || '-';
+			this.refreshDisplayedProperties();
+		}
+	}
+
+	private async handleMiaaStatusUpdated(status: HybridSqlNsNameGetResponse): Promise<void> {
+		this._instanceProperties.status = status.status || '-';
+		this.refreshDisplayedProperties();
+	}
+
+	private handleEndpointsUpdated(endpoints: EndpointModel[]): void {
+		const kibanaQuery = `kubernetes_namespace:"${this._miaaModel.namespace}" and instance_name :"${this._miaaModel.name}"`;
+		const kibanaUrl = `${endpoints.find(e => e.name === 'logsui')?.endpoint}/app/kibana#/discover?_a=(query:(language:kuery,query:'${kibanaQuery}'))`;
+		this._kibanaLink.label = kibanaUrl;
+		this._kibanaLink.url = kibanaUrl;
+
+		const grafanaUrl = `${endpoints.find(e => e.name === 'metricsui')?.endpoint}/d/wZx3OUdmz/azure-sql-db-managed-instance-metrics?&var-hostname=${this._miaaModel.name}-0`;
```

Comment:
Nitpick.  This probably works fine but I wouldn't have put & after the ?.  It could just be ?var-hostname